lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -87,6 +88,11 @@ def configure(conf)
     if @template_name && @template_file
       template_install(@template_name, @template_file)
     end
+
+    if @templates
```

Comment:
If you do so (either ... or) then the question arises:
What to give priority when there are both options? (Or just throw an error?)